Source/Synthesis/FieldTypes/DateTimeField.cs

```diff
@@ -21,9 +21,10 @@ public virtual DateTime Value
 			{
 				if (!IsFieldLoaded && InnerSearchValue != null)
 				{
-					var converter = new IndexFieldDateTimeValueConverter();
```

Comment:
As long as this is tested with lucene and solr and works (check for time zone parsing as well), this is cool. Remove the commented lines though.